sdk/cosmosdb/Azure.ResourceManager.CosmosDB/tests/ScenarioTests/SqlResourcesOperationsTests.cs

```diff
@@ -0,0 +1,290 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Core.TestFramework;
+using Azure.ResourceManager.CosmosDB.Models;
+using NUnit;
+using NUnit.Framework;
+
+namespace Azure.ResourceManager.CosmosDB.Tests
+{
+    public class SqlResourcesOperationsTests: CosmosDBManagementClientBase
+    {
+        private string location;
+
+        // using an existing DB account, since Account provisioning takes 10-15 minutes
+        private string resourceGroupName;
+        private string databaseAccountName;
+
+        private string databaseName;
+        private string databaseName2;
+        private string containerName;
+        private string storedProcedureName;
+        private string triggerName ;
+        private string userDefinedFunctionName ;
+
+        private string sqlThroughputType;
+
+        private int sampleThroughput;
+
+        private Dictionary<string, string> additionalProperties = new Dictionary<string, string>
+        {
+            {"foo","bar" }
+        };
+        private Dictionary<string, string> tags = new Dictionary<string, string>
+        {
+            {"key3","value3"},
+            {"key4","value4"}
+        };
+        public SqlResourcesOperationsTests(bool isAsync) : base(isAsync)
+        {
+
+        }
+        private void GenerateRandomVariables()
```

Comment:
Some are not random values, the function name is a little confusing. What about "GenerateSampleValues"?